ui/tournament/css/_calendar.scss

```diff
@@ -147,3 +147,7 @@
     }
   }
 }
+
+#main-wrap {
+  --main-max-width: min(90vw, 100vw - 140px);
+}
```

Comment:
Because you missed closing line. I open a PR you can merge it: https://github.com/randompearl-stack/lila/pull/1

Reply:
Doesnt `100vw - 140px` need to be `calc(100vw - 140px)`

Reply:
> Because you missed closing line

Can you please explain what a closing line is? I looked at your edit and couldn't tell what changed. 😅

Reply:
> Doesnt `100vw - 140px` need to be `calc(100vw - 140px)`

I think you're right because they are different units, but the browsers I tested with calculated the length without the calc.